datasette/static/datasette-manager.js

```diff
@@ -0,0 +1,202 @@
+// Custom events for use with the native CustomEvent API
+const DATASETTE_EVENTS = {
+  INIT: "InitDatasette", // returns datasette manager instance in evt.detail
```

Comment:
I'm flexible, I saw a bunch of opinions about this here:

https://stackoverflow.com/questions/19071579/javascript-dom-event-name-convention

It looks like it might be safe to err on the side of all-lowercase. `datasette_init` or `datasette.init` both have a reasonable look to them.